README.md

```diff
@@ -3,9 +3,11 @@ KestrelHttpServer
 
 [![Join the chat at https://gitter.im/aspnet/KestrelHttpServer](https://badges.gitter.im/aspnet/KestrelHttpServer.svg)](https://gitter.im/aspnet/KestrelHttpServer?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-AppVeyor: [![AppVeyor](https://ci.appveyor.com/api/projects/status/nr0s92ykm57q0bjv/branch/dev?svg=true)](https://ci.appveyor.com/project/aspnetci/KestrelHttpServer/branch/dev)
+[![Travis build status][travis-badge]](https://travis-ci.org/aspnet/KestrelHttpServer/branches)
```

Comment:
> I prefer showing the main/dev branch like we do for our appveyor link.

Actually, AppVeyor is the same as Travis in this regard. It shows the "latest" build, which may include PRs. For example, if you click on the appveyor badge for SignalR right now, you see red even though the "dev" branch is passing.

![image](https://user-images.githubusercontent.com/2696087/29429069-30f4a8a4-8344-11e7-8d2b-0839085a693a.png)

![image](https://user-images.githubusercontent.com/2696087/29429108-506e76ce-8344-11e7-9ead-efdd9c2c08d4.png)

This is why I changed the link to "/branches": failing PR's aren't the first thing people will see when they click the link.

![image](https://user-images.githubusercontent.com/2696087/29429175-953ffb6a-8344-11e7-845d-9effb7bfd342.png)